No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda'
07/23/2024 11:49:19 - INFO - __main__ -   Namespace(model_type='roberta', model_name_or_path='microsoft/codebert-base', output_dir='model/java', load_model_path='model/java/checkpoint-best-bleu/pytorch_model.bin', train_filename=None, dev_filename=None, test_filename='../dataset/java/test.jsonl', config_name='', tokenizer_name='', max_source_length=256, max_target_length=128, do_train=False, do_eval=False, do_test=True, do_lower_case=False, no_cuda=True, quantize=False, quantize4=True, quantizef8=False, prune=False, prune4=False, prune6=False, do_retrain=False, train_batch_size=8, eval_batch_size=64, gradient_accumulation_steps=1, learning_rate=5e-05, beam_size=10, weight_decay=0.0, adam_epsilon=1e-08, max_grad_norm=1.0, num_train_epochs=3, max_steps=-1, eval_steps=-1, train_steps=-1, warmup_steps=0, local_rank=-1, seed=42, job_id='12205', file_len=300)
07/23/2024 11:49:19 - WARNING - __main__ -   Process rank: -1, device: cpu, n_gpu: 2, distributed training: False
07/23/2024 11:49:28 - INFO - __main__ -   reload model from model/java/checkpoint-best-bleu/pytorch_model.bin
07/23/2024 11:49:37 - INFO - __main__ -   Apply quantization qint4
07/23/2024 11:49:38 - INFO - __main__ -   ******* Calibration **************
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
  0%|          | 0/172 [00:00<?, ?it/s]slurmstepd-compute-0-3: error: *** STEP 12205.0 ON compute-0-3 CANCELLED AT 2024-07-23T13:19:37 ***
slurmstepd-compute-0-3: error: *** JOB 12205 ON compute-0-3 CANCELLED AT 2024-07-23T13:19:37 ***
